test/firrtl/lower-to-rtl-module.mlir

```diff
@@ -202,12 +199,7 @@
     // CHECK: [[OUTE:%.+]] = firrtl.pad [[INE]], 4 : (!firrtl.uint<3>) -> !firrtl.uint<4>
     firrtl.connect %outE, %inE : !firrtl.flip<uint<4>>, !firrtl.uint<3>
 
-    // Truncation for inF
```

Comment:
I took a look at the [commit](https://github.com/llvm/circt/commit/66da348c3bccf9431179d17b9f975b7862da25a8) where this was originally added, and concluded this was never really valid for `firrtl.connect`. The spec does mention automatic truncation for _partial_ connects:

> Partial connect statements from a wider ground
type component to a narrower ground type component will have its value
automatically truncated to fit the smaller bit width.

In this case, I don't see how this test case can be valid, so I just removed it. Maybe I am missing something?